virtual_workstation_setup.sh

```diff
@@ -12,6 +12,27 @@ silent_grep() {
     command grep -q  > /dev/null 2>&1 "$@"
 }
 
+set +e
+for filename in ~/.ssh/id_*; do
+    [[ "$filename" == *".pub" ]] && continue
+    # check if SSH key has passphrase
+    ssh-keygen -y -P "" -f "$filename" > /dev/null 2>&1
+    [ $? -eq 0 ] && continue
+    # check if ssh agent is running and have any SSH keys added
+    # (we assume that the key with passphrase will be added, but not checking it explicitly)
+    ssh-add -l > /dev/null 2>&1
+    if [ $? -ne 0 ]; then
+        echo "SSH key with passphare found: $filename"
+        echo "Please do the following steps:"
+        echo " - Setup ssh-agent"
+        echo " - Add SSH key with passphrase to the ssh-agent"
+        echo " - Re-run the command"
```

Comment:
💅  "Rerun the Evergreen project setup command"

Reply:
done